apps/opencs/view/render/instancemode.hpp

```diff
@@ -29,13 +33,22 @@ namespace CSVRender
                 DragMode_Scale
             };
 
+            enum DropMode
+            {
+                Collision,
+                Terrain,
+                Collision_sep,
+                Terrain_sep
+            };
```

Comment:
This doesn't quite match our naming scheme. CollisionSep instead of Collision_sep. Or Collision_Sep, if you absolutely must.

Also, we normally prefix the individual enum names with the name of the enum itself (DropMode_Collision). Alternatively you may use a class enum, which would then require explicitly stating the enum on use (DropMode::Collision).